pkg/views/workspace/info/view.go

```diff
@@ -195,3 +199,10 @@ func getInfoLineGitStatus(key string, status *apiclient.GitStatus) string {
 
 	return output
 }
+
+func getInfoLinePrNumber(PrNumber *int32, repo apiclient.GitRepository, state *apiclient.ProjectState) string {
+	if PrNumber != nil && (state == nil || state.GitStatus.CurrentBranch == repo.Branch) {
+		return getInfoLine("PR Number", fmt.Sprintf("%d", *PrNumber)) + "\n"
```

Comment:
Let's add a `#` at the start and we're good to go. (`fmt.Sprintf("#%d", *PrNumber)`)